engine/storage_mongo_datadb.go

```
@@ -1969,12 +1969,12 @@ func (ms *MongoStorage) SetFilterIndexesDrv(dbKey string, indexes map[string]uti
 	return
 }
 
-func (ms *MongoStorage) RemoveFilterIndexesDrv(id string) (err error) {
+func (ms *MongoStorage) RemoveFilterIndexesDrv(dbKey string) (err error) {
 	session, col := ms.conn(colRFI)
 	defer session.Close()
-	err = col.Remove(bson.M{"key": id})
+	err = col.Remove(bson.M{"key": dbKey})
 	if err == mgo.ErrNotFound {
-		err = utils.ErrNotFound
+		err = nil
```

please don't mask errors in the driver, the initial approach was the right one